client/widgets/test-run-state/state.directive.js

```diff
@@ -0,0 +1,184 @@
+angular.module('probedock.testRunStateWidget').directive('testRunStateWidget', function() {
```

Comment:
I don't like the name very much. It sounds like this is a widget to show the state of one test run, but it does much more than that since it displays the project health. It's the same code that you extracted from the project health widget.

I would suggest two alternatives:

1) Put the code back in the project health widget and add a scope argument that can be used to switch between the full template with a Bootstrap panel, or the simplified template (the full template should be able to include the simplified one with an `ng-include`, avoiding duplication).

2) Rename this widget `compactProjectHealthWidget` or another similar prefix like "simple" or "bare".

What do you think?